src/Utilities/RecoilAtoms.res

```diff
@@ -75,6 +75,7 @@ let userPixCPF = Recoil.atom("userPixCPF", defaultFieldValues)
 let userPixCNPJ = Recoil.atom("userPixCNPJ", defaultFieldValues)
 let isCompleteCallbackUsed = Recoil.atom("isCompleteCallbackUsed", false)
 let isPaymentButtonHandlerProvidedAtom = Recoil.atom("isPaymentButtonHandlerProvidedAtom", false)
+let sdkScreenType = Recoil.atom("sdkScreenType", defaultScreenValues)
```

Comment:
We already have an atom for this named as `showCardFieldsAtom` can you use the same as it will increase complexity in maintaining two same things.

Reply:
done